Source/WTF/wtf/glib/SysprofAnnotator.h

```diff
@@ -0,0 +1,456 @@
+        case WebHTMLViewPaintEnd:
+            return "WebHTMLViewPaint";
+
+        case BackingStoreFlushStart:
+        case BackingStoreFlushEnd:
+            return "BackingStoreFlush";
+        case BuildTransactionStart:
+        case BuildTransactionEnd:
+            return "BuildTransaction";
+        case SyncMessageStart:
+        case SyncMessageEnd:
+            return "SyncMessage";
+        case SyncTouchEventStart:
+        case SyncTouchEventEnd:
+            return "SyncTouchEvent";
+        case InitializeWebProcessStart:
+        case InitializeWebProcessEnd:
+            return "InitializeWebProcess";
+        case RenderingUpdateRunLoopObserverStart:
+        case RenderingUpdateRunLoopObserverEnd:
+            return "RenderingUpdateRunLoopObserver";
+        case LayerTreeFreezeStart:
+        case LayerTreeFreezeEnd:
+            return "LayerTreeFreeze";
+        case FlushRemoteImageBufferStart:
+        case FlushRemoteImageBufferEnd:
+            return "FlushRemoteImageBuffer";
+        case CreateInjectedBundleStart:
+        case CreateInjectedBundleEnd:
+            return "CreateInjectedBundle";
+        case PaintSnapshotStart:
+        case PaintSnapshotEnd:
+            return "PaintSnapshot";
+        case RenderServerSnapshotStart:
+        case RenderServerSnapshotEnd:
+            return "RenderServerSnapshot";
+        case TakeSnapshotStart:
+        case TakeSnapshotEnd:
+            return "TakeSnapshot";
+        case SyntheticMomentumStart:
+        case SyntheticMomentumEnd:
+            return "SyntheticMomentum";
+        case SyntheticMomentumEvent:
+            return "SyntheticMomentumEvent";
+        case RemoteLayerTreeScheduleRenderingUpdate:
+            return "RemoteLayerTreeScheduleRenderingUpdate";
+
+        case CommitLayerTreeStart:
+        case CommitLayerTreeEnd:
+            return "CommitLayerTree";
+        case ProcessLaunchStart:
+        case ProcessLaunchEnd:
+            return "ProcessLaunch";
+        case InitializeSandboxStart:
+        case InitializeSandboxEnd:
+            return "InitializeSandbox";
+        case WebXRCPFrameWaitStart:
+        case WebXRCPFrameWaitEnd:
+            return "WebXRCPFrameWait";
+        case WebXRCPFrameStartSubmissionStart:
+        case WebXRCPFrameStartSubmissionEnd:
+            return "WebXRCPFrameStartSubmission";
+        case WebXRCPFrameEndSubmissionStart:
+        case WebXRCPFrameEndSubmissionEnd:
+            return "WebXRCPFrameEndSubmission";
+
+        case WakeUpAndApplyDisplayListStart:
+        case WakeUpAndApplyDisplayListEnd:
+            return "WakeUpAndApplyDisplayList";
+
+
+        case WTFRange:
+        case JavaScriptRange:
+        case WebCoreRange:
+        case WebKitRange:
+        case WebKit2Range:
+        case UIProcessRange:
+        case GPUProcessRange:
+        default:
+            return nullptr;
+        }
+    }
+
+    CString m_processName { "WebKit" };
```

Comment:
Can this be used without initialization? It also calls the sysprof init function, so I guess it can't. It should be `ASCIILiteral` too instead of `CString`.